src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Icu.cs

```diff
@@ -226,6 +222,18 @@ private static string IcuGetLanguageDisplayName(string cultureName)
             return null;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static void IcuIsEnsurePredefinedLocaleName(string name)
+        {
+            Debug.Assert(!GlobalizationMode.UseNls);
+
+            if (!Interop.Globalization.IsPredefinedLocale(name))
+            {
+                throw new CultureNotFoundException(nameof(name), SR.Format(SR.Argument_InvalidPredefinedCultureName, name),
+                    message: "View https://docs.microsoft.com/en-us/aspnet/core/blazor/globalization-localization?view=aspnetcore-5.0#blazor-webassembly to load a new culture");
```

Comment:
>message [](start = 20, length = 7)

the exception thrown here os not specific to wasm. so please don't use a message specific to blazor. if it is really needed, then condition the message to be added only if running in wasm.